app/routes/u.github/update-profile.ts

```diff
@@ -0,0 +1,37 @@
+import { ActionFunction, redirect } from "@remix-run/node";
+import { updateProfileForUser } from "~/database.server";
+import { getCurrentUser, getSession } from "~/session.server";
+import { UserProfile } from "~/types";
+import { getProfileRouteForUser } from "~/utils/routes";
+
+export const action: ActionFunction = async ({ request }) => {
+  const session = await getSession(request.headers.get("Cookie"));
+  const currentUser = await getCurrentUser(session);
+
+  if (!currentUser) {
+    throw new Error("You must be logged in to save your profile");
+  }
+
+  const formData = await request.formData();
+
+  const twitterUrl = formData.get("twitter")?.toString() ?? "";
+  const linkedinUrl = formData.get("linkedin")?.toString() ?? "";
+
+  // TODO
+  // const techInterests = formData.get("techInterests");
+  // const subjectInterests = formData.get("subjectInterests");
+  // const contributionInterests = formData.get("contributionInterests");
+  // const joinDiscord = formData.get("joinDiscord");
```

Comment:
Note: Just double checking that these comments are here on purpose. Maybe attach a ticket?

Reply:
They are! Tracking this in [ENG-570](https://linear.app/codesee/issue/ENG-570/save-users-interests-on-their-profiles).